less/index.less

```diff
@@ -53,7 +53,253 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+// Variable
+@variable-nav-color: rgb(94, 94, 94);
+@variable-H1-bold: bold;
+
+// Mixin
+.flexbox-space-between-mixin {
+	justify-content: space-between;
+	display: flex;
+}
+.flexbox-row {
+	display: flex;
+	flex-direction: row;
+}
+
+h2 {
+	font-weight: @variable-H1-bold;
+	
+}
+
+// Container
+.container {
+    width: 878px;
+	margin: 0 auto;
+	/* border: 1px solid red; */
+	
+	@media (max-width: 878px){
+		width: 100%;
+		background: grey;
+	}
+}
+
+
+.navigation {
+	display: flex;
+}
+
+.navigation nav {
+	margin-top: 40px;
+	width: 610px;
+	.flexbox-space-between-mixin;
+	/* Top Right Bottom Left */
+}
+
```

Comment:
With a less compiler, although you can write your css this way, the magic of the compiler is that you can next is.  So instead of writing ( .navigation nav ) you can just nest the nav inside the .navigation on line 88.